pvlib/iotools/__init__.py

```diff
@@ -5,3 +5,5 @@
 from pvlib.iotools.surfrad import read_surfrad  # noqa: F401
 from pvlib.iotools.midc import read_midc  # noqa: F401
 from pvlib.iotools.midc import read_midc_raw_data_from_nrel  # noqa: F401
+from pvlib.iotools.ecmwf_macc import read_ecmwf_macc  # noqa: F401
+from pvlib.iotools.ecmwf_macc import get_ecmwf_macc  # noqa: F401
```

Comment:
Except that `get_ecmwf_macc()` doesn't read the data into a dataframe, so in that sense it would be different - the ECMWF api client output is a file, and it can take a while, last night it took 35 minutes
![image](https://user-images.githubusercontent.com/1385621/49967358-74744200-fed7-11e8-9e5d-77497ae16d63.png)
so I'm concerned b/c it seems fragile and error prone to me to have a function that opens a file depending on the ECMWF api client's success, over a timespan that could be hours  - therefore I would prefer to decouple the steps: (1) first get data and (2) then read data. Hopefully okay?

Reply:
I prefer `get_xxxx` when transferring data from an external source, and `read_xxxx` when parsing data from a local source.

Reply:
@mikofski I was only commenting on the naming pattern. I agree with your approach overall. I suggest leaving it as is and debating `iotools` naming conventions elsewhere.